build.gradle

```diff
@@ -10,7 +10,7 @@ plugins {
 subprojects {
 
     group = 'edu.wisc.my.restproxy'
-    version = '2.1.0'
+    version = '2.1.1'
```

Comment:
That's true, although I think it's a Bad Idea to have two revisions in source control with the same version, as a general rule (snapshots being the exception), and numbers are pretty cheap anyway.
Onward and upward!